src/components/Builder/BasicInfo/index.jsx

```diff
@@ -115,18 +135,18 @@ const BasicInfo = ({ profileData }) => {
         <Col span={12}>
           <Form.Item
             name="years_of_experience"
-            label="Years Of Experience"
+            label="Past Years Of Experience(In months)"
             rules={[
               { required: true, message: "Experience required" },
-              // {
-              //   pattern: /^[0-9]+$/,
-              //   message: "Experience must be a whole number",
-              // },
+              {
+                pattern: /^[0-9]+$/,
+                message: "Experience must be a whole number",
+              },
               {
                 validator: (_, value) =>
```

Comment:
remove validator functions whereever your have used and add simple rules instead of it. You can pass customer message for rules as well. check and library for examples.

Reply:
Done!